### PR TITLE
3.5.1 changelog updates

### PR DESCRIPTION
**Pull request by esethna**
@hmhealey please review?

### CODE REVIEW

File: source/administration/changelog.md [hunk 1]
```diff
@@ -2,13 +2,28 @@
...
-Release date: 2016-11-16
+## Release v3.5.1  
+
+### Notes on Patch Release
+
+ - **v3.5.1, released 2016-11-23**
+   - Security update to preventing cross-site scripting and remote code execution, thanks to Harrison Healey for [reporting responsibly](http://www.mattermost.org/responsible-disclosure-policy/).
+   - Fixed an issue where usernames would sometimes not appear beside posts and the reply arrow would throw an error. 
+   - The channel purpose is no longer cut off in the user interface of the **More...** channel menu. 
+   - Fix scroll issue where the center channel didn't always scroll to the bottom when switching channels.
```
Comment (jasonblais) [inline source/administration/changelog.md]: "Fixed a scrolling issue..."

File: source/administration/changelog.md [hunk 2]
```diff
@@ -2,13 +2,28 @@
...
+   - Fix wildcard searching using an asterisk.
```
Comment (jasonblais) [inline source/administration/changelog.md]: "Fixed..."

File: source/administration/changelog.md [hunk 3]
```diff
@@ -2,13 +2,28 @@
...
+   - Fixed an issue where direct message channel headers would sometimes disappear.
+   - "New Messages" indicator is fixed so it no longer remains visible after switching channels.
+   - Fixed an issue where users could not join a public channel by navigating to the channel URL.
+   - Switching teams no longer causes user interface and console errors.
```
Comment (jasonblais) [inline source/administration/changelog.md]: Was this in 3.5? Or only in one of the RCs for 3.5.1?

File: source/administration/changelog.md (hunk 2 above)
Comment (hmhealey) [inline source/administration/changelog.md]: Maybe change this to
```
- Fixed a server error that occurred when searching for users using an asterisk
```
just to make it clear that searching for posts wasn't affected.

File: source/administration/changelog.md (hunk 3 above)
Comment (hmhealey) [inline source/administration/changelog.md]: Yeah, that was added in one of the RCs so it can be removed from the changelog

Comment (esethna) [inline source/administration/changelog.md]: Updated, thanks

**Pull request merged**